### PR TITLE
Nsng 396 api documentation

### PR DESCRIPTION
**Pull request by igor-kamil**
API is generated using [Scribe](https://github.com/knuckleswtf/scribe/)

Snímka obrazovky 2023-11-03 o 13 06 15

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -32,10 +32,28 @@
...
+ * @response status=404 scenario="Code not found" {"message": "Record not found."}
+ */
 Route::get('/verify/{code}', function ($code) {
     return new CodeResource(Code::where('code', $code)->firstOrFail());
 });
 
+/**
+ * GET api/items/{id}
+ *
+ * Gets the item with the given ID. If it is found, the response will contain the item's data.
```
Comment (FrantisekMichalSebestyen) [inline routes/api.php:53]: Get the item for the given ID

File: routes/api.php [hunk 2]
```diff
@@ -86,21 +118,49 @@
...
+ */
 Route::get('/collections/{hashid}', function ($hashid) {
     $collection = Collection::findByHashidOrFail($hashid);
     return $collection->items;
 });
 
+/**
+ * GET api/stories/{id}
+ *
+ * Gets the story with the given ID. If it is, the response will contain the story's data.
```
Comment (FrantisekMichalSebestyen) [inline routes/api.php:136]: Get the story for the given ID

File: routes/api.php [hunk 3]
```diff
@@ -86,21 +118,49 @@
...
+ */
 Route::get('/stories/{id}', function (string $id) {
     $story = Story::findOrFail($id);
     return new StoryResource($story);
 });
 
+/**
+ * GET api/places/{id}
+ *
+ * Gets the place with the given ID. If it is, the response will contain the place's data.
```
Comment (FrantisekMichalSebestyen) [inline routes/api.php:148]: Get the place for the given ID

File: routes/api.php (hunk 2 above)
Comment (FrantisekMichalSebestyen) [reply routes/api.php:136]: No need for rest

File: routes/api.php [hunk 4]
```diff
@@ -86,21 +118,49 @@
...
+ */
 Route::get('/places/{id}', function (string $id) {
     $place = Place::findOrFail($id);
     return new PlaceResource($place);
 });
 
+/**
+ * GET api/bucketlists/{id}
+ *
+ * Gets the items for the bucketlist/scavenger hunt. Returns the list of items and their data.
```
Comment (FrantisekMichalSebestyen) [inline routes/api.php:160]: Get instead of gets

File: routes/api.php [hunk 5]
```diff
@@ -119,6 +179,13 @@
     ]);
 });
 
+/**
+ * GET api/related_items/{ids}
+ *
+ * Gets the data for the related items. Returns the list of items and their data.
```
Comment (FrantisekMichalSebestyen) [inline routes/api.php:185]: Get instead of gets

File: public/docs/collection.json [hunk 6]
```diff
@@ -10,7 +10,7 @@
     ],
     "info": {
         "name": "Atlas SNG",
-        "_postman_id": "cad9821a-772a-4b18-94fb-25798042566b",
+        "_postman_id": "bee1fd43-2a21-4b78-b350-b89ca2c4b6ec",
```
[the remark this answers is not in the document]
Comment (igor-kamil) [reply public/docs/collection.json:13]: nop. this is just stamp AFAIK

**Review by FrantisekMichalSebestyen — APPROVED** [review]

**Pull request merged**